privacy_evaluator/datasets/cifar10.py

```diff
@@ -78,12 +80,27 @@ def pytorch_loader(
         )
 
     @classmethod
-    def numpy(cls, one_hot_encode: bool = True) -> Tuple[np.ndarray, ...]:
-        """Loads train and test dataset as a numpy arrays.
+    def numpy(
+        cls, model_type: str, one_hot_encode: bool = True
+    ) -> Tuple[np.ndarray, ...]:
+        """Loads train and test dataset for given model type as a numpy arrays.
 
         :param one_hot_encode: If data should be one-hot-encoded.
```

Comment:
This is a good parameter, and I would like to see that question taken into consideration more in other classes as well.